pkg/security/resolvers/dentry/resolver.go

```diff
@@ -803,10 +768,15 @@ func NewResolver(config *config.Config, statsdClient statsd.ClientInterface, e *
 		return nil, fmt.Errorf("couldn't fetch the host CPU count: %w", err)
 	}
 
+	cache, err := cache.NewTwoLayersLRU[uint32, model.PathKey, PathEntry](8000)
```

Comment:
Should we reuse the `config.DentryCacheSize` parameter to initialize the cache?
Re-using isn't really forward-compatible because the behavior isn't the same anymore but because the cache size is now shared across multiple mount IDs it should not be a problem

Reply:
Maybe not, the issue real issue is that it isn't backward compatible (i.e. using a high cache size with this version and then reverting to an older version will cause memory usage to increase)